myapp/backend/backend.py

```diff
@@ -34,12 +37,149 @@ def remove(self):
       return resp.deleted_count
 
 class User(Model):
-  client = pymongo.MongoClient("mongodb+srv://TeamProjAdmin:teamproject308@cluster0.3xlma.mongodb.net/TeamProj?retryWrites=true&w=majority&tls=true&tlsAllowInvalidCertificates=true")
-  # db = client.TeamProj
-  collection = client.TeamProj.TeamRoster
+  #client = pymongo.MongoClient("mongodb+srv://TeamProjAdmin:teamproject308@cluster0.3xlma.mongodb.net/TeamProj?retryWrites=true&w=majority&tls=true&tlsAllowInvalidCertificates=true")
```

Comment:
looks good. I think we changed it in the main branch to separate our the various components, so we can work on that with merging.